charts/minio/templates/deployment.yaml

```diff
@@ -108,6 +116,18 @@ spec:
       - name: minio-backup
         emptyDir: {}
       {{- end }}
+      {{- if .Values.minio.certificatesSecret }}
+      - name: certificates-volume
+        secret:
+          secretName: {{ .Values.minio.certificatesSecret }}
+          items:
+          - key: tls.crt
+            path: public.crt
+          - key: tls.key
+            path: private.key
+          - key: tls.crt
+            path: CAs/public.crt
```

Comment:
Why is the cert mounted twice?

Reply:
needs to go to two places: https://github.com/minio/minio/tree/master/docs/tls/kubernetes#3-update-deployment-yaml-file